modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React, {useContext, useState} from 'react';
+
+import {SidebarContext} from '../sidebar/SidebarContext.es';
+
+export default ({fields}) => {
+	const [itemSelected, setItemSelected] = useState(null);
+	const {portletNamespace} = useContext(SidebarContext);
+
+	const shortcuts = fields.map((field, index) => (
+		<li key={`item-${index}`} onClick={() => setItemSelected(index)}>
```

Comment:
@alinedoleron, is `item` the same thing than `card-item`? If so, would be important we keep the same naming along the code.

Besides, the state `itemSelected` holds the index of the item selected, in light of this we should rename the state to reflect it, maybe `itemSelectedIndex`. What do you think?